src/SFA.DAS.EmployerFinance.Messages/DummyEvent.cs

```diff
@@ -0,0 +1,9 @@
+using NServiceBus;
+
+namespace SFA.DAS.EmployerFinance.Messages
+{
+    public class DummyEvent : IEvent
```

Comment:
This message can be removed if you prefer, we'll verify that any communication is working as part of the health check ticket.